wave_lang/kernel/ops/wave_ops.py

```diff
@@ -778,6 +786,10 @@ def type(self, value: Any):
     def has_side_effects(self) -> bool:
         return False
 
+    @property
+    def debug_log_transformer(self) -> bool:
```

Comment:
Nit: can we have some doc on what is the contract that subclasses should follow when implementing this?